scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -33,6 +33,7 @@ workflow Mutect2 {
   File tumor_bam_index
   File? normal_bam
   File? normal_bam_index
+    String vcf_name = basename(tumor_bam, ".bam") + ".vcf"
```

Comment:
Can you rename to "output_vcf_name"?

Also, not sure if this is github.com website, but this line looks oddly indented.

Reply:
I indented due to my renegade convention to indent derived variables so people know not to put them in the json, but I can undo it.

Reply:
> Can you rename to "output_vcf_name"?

done.